### PR TITLE
imolement react person

### PR DESCRIPTION
**Pull request by iHateAntichrist**
[DEMO LINK](https://iHateAntichrist.github.io/react_person/)

### CODE REVIEW

File: src/components/Person/Person.jsx [hunk 1]
```diff
@@ -1 +1,34 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  let partner = '';
```
Comment (maksym-didyk) [inline src/components/Person/Person.jsx:5]: maybe better way to use ternary operator? 
Also use const instead let (this variable never changes)

**Review by maksym-didyk — APPROVED** [review]
Good job.
See my comment about const/let and if/else.

File: src/components/Person/Person.jsx (hunk 1 above)
Comment (SerhiiKirik) [reply src/components/Person/Person.jsx:5]: agree, use ternary here

File: src/components/Person/Person.jsx [hunk 2]
```diff
@@ -1 +1,34 @@
...
+        {`My name is ${name}`}
+      </h2>
+
+      { age
+      && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )
+      }
```
Comment (SerhiiKirik) [inline src/components/Person/Person.jsx:25]:
```suggestion
      {age && (
        <p className="Person__age">
          {`I am ${age}`}
        </p>
      )}
```

**Review by SerhiiKirik — CHANGES_REQUESTED** [review]
Well done 🔥 
Let's make some improvements 🧑‍💻

File: src/components/Person/Person.jsx [hunk 3]
```diff
@@ -1 +1,28 @@
...
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      { age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )
+      }
```
Comment (SerhiiKirik) [inline src/components/Person/Person.jsx:19]:
```suggestion
      { age && (
        <p className="Person__age">
          {`I am ${age}`}
        </p>
      )}
```

**Review by SerhiiKirik — APPROVED** [review]
Well done 🔥

**Review by SerhiiKirik — APPROVED** [review]
Sync 🌍

**Pull request closed without merging**